ada_moveit/calib/manual/calib_camera_pose.launch.py

```diff
@@ -0,0 +1,32 @@
+""" Static transform publisher acquired via MoveIt 2 hand-eye calibration """
+""" EYE-IN-HAND: j2n6s200_end_effector -> camera_color_optical_frame """
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description() -> LaunchDescription:
```

Comment:
Maybe add a comment that this was hand-tweaked, assuming 0 pitch and -3.14 yaw (i.e. assuming that the only unknown orientation was pitch), by having the camera observe the robot itself (i.e. pointing at Joint 1) and comparing the depth cloud to the mesh in RViz.